components/motor/roboclaw/roboclaw.go

```diff
@@ -318,7 +319,40 @@ func (m *roboclawMotor) GoTo(ctx context.Context, rpm, positionRevolutions float
 }
 
 func (m *roboclawMotor) SetRPM(ctx context.Context, rpm float64, extra map[string]interface{}) error {
-	return motor.NewSetRPMUnsupportedError(m.Name().ShortName())
+	if m.conf.TicksPerRotation == 0 {
```

Comment:
There a huge bug in this driver, where if the ticks per rotation is zero, this motor is actually not position reporting, please make a follow up ticket.

Once we have a valid Proeprties function we can call that to check what power percentage to calculate in each case within this driver.